src/devices_models/devices/AC_branches.jl

```diff
@@ -224,7 +224,16 @@ function branch_flow_constraints!(
     ::Union{Type{PM.DCPPowerModel}, Type{StandardPTDFModel}},
     feedforward::Union{Nothing, AbstractAffectFeedForward},
 )
-    flow_range_data = [(PSY.get_name(h), PSY.get_flowlimits(h)) for h in devices]
+    flow_range_data = Vector{PSI.DeviceRange}()
+    for h in devices
+        flow_range_data = vcat(
```

Comment:
We need to do something here better than vcat, this is too slow and memory consuming when we have >1000 lines it will cause too many allocations